plugins/woocommerce/client/admin/client/settings-payments/components/bank-accounts-list/bank-accounts-list.tsx

```diff
@@ -0,0 +1,204 @@
+/**
+ * External dependencies
+ */
+import { Button, MenuGroup, MenuItem, Modal } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { EllipsisMenu } from '@woocommerce/components';
+import { useState } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { BankAccount } from './types';
+import { BankAccountModal } from './bank-account-modal';
+import {
+	DefaultDragHandle,
+	SortableContainer,
+	SortableItem,
+} from '~/settings-payments/components/sortable';
+import './bank-account-list.scss';
+
+function generateId() {
```

Comment:
Suggestion: add comment about what ID we are generating here 🙂

Reply:
This code will end up getting removed when I move onto implementing this in the offline methods, as we'll rely on the internal IDs 🙂

Reply:
Gotcha, thanks!